### PR TITLE
Fixed python latest version metadata

### PR DESCRIPTION
**Pull request by gerrod3**
[noissue]

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -16,32 +16,32 @@ RUN dnf -yq install \
...
+# RUN make -C pulp-fixtures all-debian
 
 # === Serve content ===========================================================
 FROM registry.access.redhat.com/ubi8/ubi AS server
 
 RUN dnf install -y nginx && dnf clean all
 RUN rm /usr/share/nginx/html/index.html
 COPY --from=fedora-build pulp-fixtures/fixtures /usr/share/nginx/html
-COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
+# COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
```
Comment (mdellweg) [inline Containerfile:44]: Is this PR maybe still draft?

Comment (gerrod3) [reply Containerfile:44]: Woops, accidentally added my modified containerfile.

**Review by mdellweg — COMMENTED** [review]
I don't understand enough of this repo format, but the code looks good.

**Review by dralley — APPROVED** [review]

**Review by mdellweg — APPROVED** [review]

**Comment by daviddavis** [comment]
Thanks @gerrod3. After this job completes, can you confirm that the changes are live at fixtures.pulpproject.org?

https://github.com/pulp/pulp-fixtures/actions/runs/686433608

**Comment by gerrod3** [comment]
The changes are live on fixtures.pulpproject.org.

**Pull request merged**